test/common/test_runner.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package common
 
 import (
+	goerrs "errors"
```

Comment:
I'd rather we alias k8s package rather than built in, but 🤷‍♂

Reply:
yeah I thought about that but decided to keep the amount of changes lower - but I'll do it